src/test/scala/afenton/bazel/bsp/LspTestProcess.scala

```diff
@@ -232,8 +236,22 @@ case class LspTestProcess(workspaceRoot: Path):
       _ <- d2.get
     yield resp
 
+  private def shutdown(
+      client: BspClient,
+      exitSwitch: Deferred[IO, Either[Throwable, Unit]]
+  ): IO[Unit] =
+    for
+      d1 <- client.buildShutdown(())
+      resp <- d1.get
```

Comment:
`resp` is never used. I think `_ <- d1.get` is what you want, you are just blocking until the d1 is completed, right?